src/components/ShopingCart/ShopingCart.jsx

```diff
@@ -0,0 +1,198 @@
+                    src={item.image}
+                    className="ml-2 w-24 md:w-28 h-24"
+                    alt={item.title}
+                    loading="lazy"
+                  />
+                </div>
+                <div className="ml-5">
+                  <div className="flex ">
+                    <Link className="text-blue-900 hover:text-blue-950 transition duration-300 ease-in-out text-sm font-semibold tracking-tight xl:text-lg">
+                      {item.title}
+                    </Link>
+                    <h2 className="absolute right-1 text-black text-base font-semibold tracking-tight ">
+                      {item.price} $
+                    </h2>
+                  </div>
+                  <h2 className="text-black text-opacity-40 text-base font-medium tracking-tight">
+                    by {item.author}
+                  </h2>
+                  <span className="text-black text-lg ">{item.rating}</span>
+                  {Array.from({ length: item.rating }, (_, index) => (
+                    <FontAwesomeIcon
+                      key={index}
+                      icon={faStar}
+                      className="text-yellow-500 ml-1"
+                    />
+                  ))}
+                  <div className="flex my-1">
+                    <h3 className="text-black text-opacity-40 text-sm font-medium tracking-tight ">
+                      <FontAwesomeIcon className="mr-1" icon={faVideo} />
+                      {item.numOfVideos} videos
+                    </h3>
+                    <h3 className="text-black hidden lg:block text-opacity-40 text-sm font-medium tracking-tight ml-5">
+                      {item.numOfLectures} Lectures
+                    </h3>
+                    <h3 className="text-black hidden lg:block text-opacity-40 text-sm font-medium tracking-tight ml-5">
+                      {item.numOfSections} Sections
+                    </h3>
+                    <button className=" absolute right-1 text-blue-700 text-base font-semibold tracking-tight hover:text-blue-950 transition duration-300 ease-in-out">
+                      <FontAwesomeIcon icon={faTimes} /> Remove
+                    </button>
+                  </div>
+                </div>
+              </div>
+            );
+          })}
+          <UserCoursesPagination
+            coursesLength={ShopingCartCourses.length}
+            getCurrentPage={getCurrentPage}
+          />
+        </div>
+        <div className="order-first md:order-none w-full md:w-[316px] h-[214px] pt-4 pl-7 ml-0 md:ml-20 2xl:ml-40 my-5  md:mt-16 bg-white rounded-[1.5rem] shadow-xl flex flex-col justify-center">
+          <h1 className="text-black text-xl font-semibold tracking-tight">
+            Total Price:
+          </h1>
+          <h2 className="text-black text-[32px] font-semibold">$200.00</h2>
+          <h3 className="text-black text-sm font-semibold font-['Open Sans'] my-2">
+            Courses in Cart: 4
```

Comment:
Here We need to render this total number of courses dynamic, so after fetching all purchased courses, you may use (ShopingCartCourses.length)

Reply:
i do this , thank you for all comments.